### ISSUE TITLE
[Accessibility][Highcharts][Angular] No way to change default aria-label on <highcharts-chart> component

### ISSUE THREAD

**Issue by karolisZuk** [highcharts/highcharts#23114]
Hello! While doing accessibility testing I ran into an issue. I am using highcharts-angular, to render multiple charts on one page. The charts are landmarks from screen reader perspective, and landmarks must have unique aria-label, or title. The default aria-label for <highcharts-chart> is set to "Chart. Highcharts interactive chart." From what I gathered in documentation, there is no way to change this aria-label (or remove it, and use title instead). If thats the case, automated accessibility tools will mark this as error, if page has multiple charts.

<img width="488" alt="Image" src="https://github.com/user-attachments/assets/1972fd81-9ad6-40b8-9a7f-0d207e9702b3" />

#### Expected behaviour
User should be able to set custom aria-label, or aria-label should be set to be same as chart title.

#### Product version
"highcharts": "^11.3.0",
"highcharts-angular": "^4.0.0",

#### Affected browser(s)
All.

**Comment by jamesonhill** [comment]
+1
